backend/battle/tests.py

```diff
@@ -1,3 +1,14 @@
-# from django.test import TestCase
+from django.test import TestCase
+from battle.battles.battle import validate_sum_pokemons
 
-# Create your tests here.
+
+class PokemonsSumTest(TestCase):
+    def test_sum(self):
+        pokemons = ['bulbasaur', 'pikachu', 'pidgeot']
+        sum_pokemons = validate_sum_pokemons(pokemons)
```

Comment:
Let's not test getting the pokemon from the API. It's not a good practice to allow tests to make requests for external APIs. Please get the pokemon's from the database

Reply:
Thanks :)